src/app/metrics/site-metrics/site-metrics.component.ts

```diff
@@ -0,0 +1,172 @@
+import { Component, OnDestroy } from '@angular/core';
+import { ToastService } from '../../services/toast.service';
+import { ApiService } from '../../services/api.service';
+import { Constants } from '../../shared/utils/constants';
+import { firstValueFrom, Subscription } from 'rxjs';
+import { DataService } from '../../services/data.service';
+import { LoadingService } from '../../services/loading.service';
+
+@Component({
+  selector: 'app-site-metrics',
+  templateUrl: './site-metrics.component.html',
+  styleUrls: ['./site-metrics.component.scss'],
+})
+export class SiteMetricsComponent implements OnDestroy {
+  private subscriptions = new Subscription();
+  public loading: boolean = false;
+
+  parkName: string;
+  facilityName: string;
+  passStatusChart;
+  passExport;
+  isGenerating: any = false;
+  statusMessage: any;
+  signedURL: any;
+  buttonText: any = 'Export Pass Data';
+  doughnutLabels: any;
+  doughnutData: any;
+  doughnutOptions: any = {
+    responsive: true,
+    maintainAspectRatio: false,
+    plugins: {
+      legend: {
+        labels: {
+          font: {
+            size: 20,
+          },
+        },
+      },
+    },
+  };
+  doughnutPlugins: any;
+  scatterLabels: any;
+  scatterData: any;
+  scatterOptions: any;
+  passTotal: number = 0;
+  passActive: number = 0;
+  passReserved: number = 0;
+  passExpired: number = 0;
```

Comment:
I believe these are better off initialized to `undefined` instead of `0` as it will take advantage of the `|| '-'` mode of the basic-metric.component.html:
```
<h1 class="mb-0">{{ (_value | number : "1.0-0" : "en-US") || "-" }}</h1>
```